static/app/views/performance/landing/index.tsx

```diff
@@ -69,7 +72,33 @@ export function PerformanceLanding(props: Props) {
 
   const {teams, initiallyLoaded} = useTeams({provideUserTeams: true});
 
-  const currentLandingDisplay = getCurrentLandingDisplay(location, projects, eventView);
+  const hasMounted = useRef(false);
+  const paramLandingDisplay = getLandingDisplayFromParam(location);
+  const defaultLandingDisplayForProjects = getDefaultDisplayForPlatform(
+    projects,
+    eventView
+  );
+  const [landingDisplay, setLocalLandingDisplay] = useState(
+    paramLandingDisplay || defaultLandingDisplayForProjects
+  );
+
+  useEffect(() => {
+    if (hasMounted.current) {
+      setLocalLandingDisplay(defaultLandingDisplayForProjects);
+      browserHistory.replace({
+        pathname: location.pathname,
+        query: {
+          ...omit(location.query, ['landingDisplay']),
+          landingDisplay: undefined,
+        },
```

Comment:
You dont need to add `landingDisplay` back right? so `query: omit(location.query, ['landingDisplay'])` should be enough?

Reply:
This is equivalent to just doing: 
```
        query: {
          ...location.query,
          landingDisplay: undefined,
        },
```